src/main/java/net/minestom/arena/game/mob/NextStageNPC.java

```diff
@@ -0,0 +1,41 @@
+package net.minestom.arena.game.mob;
+
+import net.kyori.adventure.sound.Sound;
+import net.minestom.arena.Messenger;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.entity.Player;
+import net.minestom.server.event.EventFilter;
+import net.minestom.server.event.EventNode;
+import net.minestom.server.event.player.PlayerEntityInteractEvent;
+import net.minestom.server.event.trait.InstanceEvent;
+import net.minestom.server.sound.SoundEvent;
+
+public class NextStageNPC extends Entity {
+    private final MobArena arena;
+    private final EventNode<InstanceEvent> node = EventNode.type("next-stage-npc", EventFilter.INSTANCE);;
```

Comment:
This listener is unnecessary, you should be able to use `Entity#eventNode`

Reply:
Doesn't work

Reply:
Fixed now